nextcord/raw_models.py

```diff
@@ -311,4 +314,21 @@ def __init__(self, data: TypingEvent) -> None:
         try:
             self.guild_id: Optional[int] = int(data['guild_id'])
         except KeyError:
-            self.guild_id: Optional[int] = None
\ No newline at end of file
+            self.guild_id: Optional[int] = None
+
+
+class RawMemberRemoveEvent(_RawReprMixin):
+    """Represents the payload for a :func:`on_raw_member_remove` event.
+    .. versionadded:: 2.0
+    Attributes
+    -----------
+    guild_id: :class:`int`
+        The guild ID where the member left from.
+    user: :class:`User`
+        The user who left the guild."""
```

Comment:
```suggestion
        The user who left the guild.
    """
```

Reply:
^ for this, https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation is the reason